maven_install.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
suggestion(non-blocking): Could you add a [linguist-generated entry](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github) for this file?

Reply:
Good idea, done.

Reply:
I removed the maven_install.json files since dependency pinning was causing CI failures and I don't think we need to pin these files, the resolution should always be stable anyways.